src/Lean/Elab/MutualDef.lean

```diff
@@ -111,9 +118,13 @@ private def getPendingMVarErrorMessage (views : Array DefView) : String :=
   | some ids =>
     let idsStr := ", ".intercalate <| ids.map fun id => s!"`{id}`"
     let paramsStr := ", ".intercalate <| ids.map fun id => s!"`({id} : _)`"
-    s!"\nrecall that you cannot declare multiple constants in a single declaration. The identifier(s) {idsStr} are being interpreted as parameters {paramsStr}"
+    -- TODO: whoever prints the 'notes' should add the line breaks.
+    s!"\nnote: recall that you cannot declare multiple constants in a single declaration. The identifier(s) {idsStr} are being interpreted as parameters {paramsStr}"
```

Comment:
Should use the new `MessageData.note` combinator, which adds whitespace before and capitalizes `Note`